x-pack/plugins/canvas/tasks/helpers/webpack.plugins.js

```diff
@@ -94,6 +94,16 @@ export function getWebpackConfig({ devtool, watch } = {}) {
           ignore: '**/__tests__/**',
         },
       ]),
+      function canvasStatsGenerator() {
+        if (!process.env.CANVAS_GENERATE_STATS) {
+          return;
+        }
+
+        this.hooks.done.tap('canvas_stats', stats => {
```

Comment:
I know that "Webpack hooks" are now a thing, is this an example of one? Does this just run when Webpack is "done" then?